lib/util/time.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Util.Time do
+  @typedoc """
+  Unix timestamp, seconds past the epoch
+  """
+  @type timestamp :: integer()
+
+  @spec current_time :: timestamp
+  def current_time do
+    System.system_time(:second)
+  end
+
+  @spec current_datetime :: DateTime.t()
+  def current_datetime do
+    DateTime.now!("America/New_York")
```

Comment:
**suggestion:** this, and the other places we reference the local time zone, could go in an application configuration variable (like `config/config.exs`).

Reply:
I'll add it to the config. But I'm afraid of removing it from `current_datetime` and other time functions, since the consequences of timezone issues are so severe...

Reply:
I'm confused - we wouldn't be removing references to time zones in general, it's just that we would point to the configuration for the particular time zone in question.

Reply:
Oh, I thought this was a global timezone override or something, akin to the TZ environment variable in some runtimes. Yeah, I'll change this.

Reply:
Ah, yeah, I don't mean some kind of global override, just a configuration value that we could directly reference whenever we need the name of a time zone for invoking some `DateTime` or `Timex` function.

Reply:
2b6e3ff